src/components/Repos.js

```diff
@@ -129,27 +162,51 @@ const Repos = ({ materiaSelected, repos, materias, partialLoading }) => {
   );
 };
 
-const SortFeature = ({ sortOption, setSortOption }) => {
+const DisplayFeature = ({ sortOption, setSortOption, languageFilter, setLanguageFilter, allLanguages }) => {
   return (
-    <Tooltip label={`Ordenado por ${sortOption.longName}`}>
-      <IconButton
-        position="absolute"
-        top="0"
-        right="0"
-        variant="link"
+    <Box
+      position="sticky"
+      top={0}
+      display="flex"
+      alignItems="center"
+      justifyContent="space-between"
+      p={4}
+      bg={useColorModeValue("purple.50", "purple.100")}
```

Comment:
```suggestion
      bg={useColorModeValue("purple.50", "purple.100")}
      pb={2}
      mb={2}
      boxShadow="0px 1px 5px rgba(0, 0, 0, 0.15)"
      clipPath="polygon(0% 0%, 100% 0%, 100% 120%, 0% 120%)"
```

No se si queda tan lindo como esperaba, pero que pensas de agregar algun borde al header, para que se sepa que es sticky? 

![image](https://github.com/user-attachments/assets/6425ab5f-e9d6-408e-984a-7cf73ea58257)

Reply:
Me gusta! Queda mucho mas estetico que como esta ahora mismo con ese sombreado. Lo agrego